Tweak toast docs (#33810)
twbs/bootstrap: site/content/docs/5.0/components/toasts.md

### Live

> Click the button the below to show a toast (positioned with our utilities in the lower right corner) that has been hidden by default with `.hide`.

Reviewer: ```suggestion
Click the button below to show a toast (positioned with our utilities in the lower right corner) that has been hidden by default with `.hide`.
```

Author: doh, embarassing to spot one but not the other...

> Click the button the below to show a toast (positioned with our utilities in the lower right corner) that has been hidden by default with `.hide`.

Reviewer 2: Shouldn't it bé "Click the button below"?

Author: yup, slipped me by

### Translucent

> Toasts are slightly translucent, so they blend over whatever they might appear over.

Reviewer: ```suggestion
Toasts are slightly translucent to blend in with what's below them.
```
